packages/p2p/package.json

```diff
@@ -34,6 +34,7 @@
         "@deriv/hooks": "^1.0.0",
         "@deriv/shared": "^1.0.0",
         "@deriv/stores": "^1.0.0",
+        "@deriv/api": "^1.0.0",
```

Comment:
@niloofar-deriv This is unused, We can remove the dependency 🙇🏻

Reply:
We are using this here 
![Uploading image.png…]()